src/en/mobile/android/development/magnification.md

```diff
@@ -0,0 +1,42 @@
+﻿---
+title: "Element Magnification"
+---
+
+# Element magnification
+
+## Enlarge texts without loss of information
+
+**Target&nbsp;:** everyone and in particular the visually impaired.  
+**When&nbsp;:** during design and development.
+
+**Description&nbsp;:** 
+
+The text size must be able to be multiplied by 2 (zoom the text to 200% from the phone's accessibility settings). At this zoom level, the presentation can be altered, but the information must remain legible (no truncated or overlapped text).
+
+The application must implement dynamic text sizes, ensure the good reactivity of the display areas to text magnification (containers that adapt to the size of their content).
+  
+To allow the "large character" option, several points must be respected during development:
+- Use a dynamic font size: the "&nbsp;sp&nbsp;". This unit, specific to Android, allows to obtain a font size according to the pixel density of the screen.  
```

Comment:
`"&nbsp;sp&nbsp;"` → "sp"
Like punctuation mentioned above, quotes in English do not need spaces within.
Please correct several instances within.